moroScript.jsx

```diff
@@ -115,6 +117,23 @@
         return rtnWord;
     }
 
+    //Process dict with count to sorted dict without count value
+    function sortAndRemoveCount(dict) {
+        var toRtn = JSON.parse(JSON.stringify(dict));
```

Comment:
This looks a bit goofy :stuck_out_tongue:, Is this just for making a deep copy of the object?

https://lodash.com/docs#cloneDeep is another option that does not look quite so silly. We import lodash in the html file so we have access to all these little utilities.

Although, if you have other reasons for verifying that it is `JSON` serializable (I think this ensures that the object is plain-old-data / doesn't have circular references, which cloneDeep might not), fine as is.